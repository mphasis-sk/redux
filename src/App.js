import React from "react";

import "./App.css";
import "bootstrap/dist/css/bootstrap.min.css";

import Users from "./component/users";

function App() {
  return (
    <div className="App">
      <Users />
    </div>
  );
}

export default App;
