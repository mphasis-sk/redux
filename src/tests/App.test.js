import { shallow } from "enzyme";
import React from "react";
import App from "../App";

test("render app component", () => {
  const wrapper = shallow(<App />);
  expect(wrapper).toMatchSnapshot();
});

test("renders learn react link", () => {
  let data = 10;
  expect(data).toBe(10);
});
