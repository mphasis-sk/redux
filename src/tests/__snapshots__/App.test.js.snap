// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`render app component 1`] = `
<div
  className="App"
>
  <Connect(Users) />
</div>
`;
